Makefile

```diff
@@ -57,31 +63,42 @@ endif
 #endif
 ifeq ($(GOARCH),ppc64le)
                 DOCKERFILE ?= Dockerfile-$(BIN)-ppc64le
+local-arch:
+	RESTIC_VERSION=$(RESTIC_VERSION) \
+        ./hack/get-restic-ppc64le.sh
```

Comment:
is there a reason we can't do the `restic` binary download within the container image build, similarly to how we're doing it for the non-Power image builds?

Reply:
1. L55-56 is added to avoid the workflow failure due to local-arch target missing while building for amd64 as in this case, it will be downloaded via Dockerfile. L155 will always trigger local-arch target and L55-56 will help us avoid any failure.

2. We cant do restic binary download within the container image as this will run the workflow on amd64 machine and it will cause exec error due to arch issues if we do it via container as the Dockerfile for power will use a ppc64le arch image.

Do let me know if any further clarification is needed on this.